src/rs.cpp

```diff
@@ -3237,6 +3237,15 @@ int rs2_parse_firmware_log(rs2_device* dev, rs2_firmware_log_message* fw_log_msg
 }
 HANDLE_EXCEPTIONS_AND_RETURN(0, dev, fw_log_msg)
 
+int rs2_get_num_of_fw_logs(rs2_device* dev, rs2_error** error) BEGIN_API_CALL
+{
+    VALIDATE_NOT_NULL(dev);
+
+    auto fw_loggerable = VALIDATE_INTERFACE(dev->device, librealsense::firmware_logger_extensions);
```

Comment:
fw_loggerable -> fw_logger
Return unsigned

Reply:
done

Reply:
fw_loggerable ?

Reply:
done - changed in multiple methods in this file